db/snapshot_impl.h

```diff
@@ -10,6 +10,7 @@
 #pragma once
 #include <vector>
 
+#include "db/dbformat.h"
```

Comment:
Shouldn't this in snapshot.h, not dbformat?

Reply:
kMaxSequenceNumber is in dbformat.h

Reply:
The includes in this file are overkill.  It should include snapshot, types, and dbformat.  No need to include db.h